presto-main/src/test/java/com/facebook/presto/sql/planner/assertions/RowExpressionVerifier.java

```diff
@@ -161,7 +161,8 @@ protected Boolean visitTryExpression(TryExpression expected, RowExpression actua
             return false;
         }
 
-        return process(expected.getInnerExpression(), ((CallExpression) actual).getArguments().get(0));
+        LambdaDefinitionExpression lambdaExpression = (LambdaDefinitionExpression) ((CallExpression) actual).getArguments().get(0);
+        return process(expected.getInnerExpression(), lambdaExpression.getBody());
```

Comment:
Wondering if we should fix this in the accept function only? Or by implementing visitLambda?

Reply:
In TRY(x/1) there is no lambda as an Expression, But when we convert to RowExpression, it is: $internal$try(() -> x/1). So we're just checking that